src/Server/src/Player.java

```diff
@@ -72,60 +77,157 @@ public void run() {
             poll();
         }
 
-        private void poll () {
+        private void poll() {
             messageProvider.sendPoll();
         }
     }
 
     // Player management
 
-
-    public int getPlayerId() {
+    /**
+     * @return DB playerID
+     */
+    int getPlayerId() {
         return playerId;
     }
 
+    /**
+     * @param playerId value got from DB
+     */
     public void setPlayerId(int playerId) {
         this.playerId = playerId;
     }
 
-    public String getPlayerName() {
+    /**
+     * @return DB nickname
+     */
+    String getPlayerName() {
         return playerName;
     }
 
+    /**
+     * @param playerName value got from DB
+     */
     public void setPlayerName(String playerName) {
         this.playerName = playerName;
     }
 
+    /**
+     *
+     */
     private void disconnect() {
         up.disconnectPlayer(this);
         Thread.currentThread().interrupt();
     }
 
+    // DB management
+
+    /**
+     * Checks for user in the database, compares passwords
+     * Sends appropriate message to the client afterwards
+     *
+     * @param message String message which method uses
+     */
+    private void handleLogin(Message message) {
+        try {
+            String splitted[] = message.getStringMultipleContent();
+            String uName = splitted[0];
+            String uPasswd = encryptionProvider.encrypt(splitted[1]);
+
+            String query = "SELECT * FROM accounts WHERE nickname = '" + uName + "'";
+            ResultSet resultSet = databaseCommunicator.executeQuery(query);
+            if (!resultSet.isBeforeFirst()) {
+                messageProvider.sendMessage(new Message("alert", "Wrong username!"));
+                return;
+            }
+            resultSet.next();
+            if (resultSet.getString("password").equals(uPasswd)) {
+                messageProvider.sendMessage(new Message("login", true));
+                setPlayerName(uName);
+                setPlayerId(resultSet.getInt("user_id"));
```

Comment:
jeśli dobrze rozumiem, to getInt już nie może tu być, bo user_id jest stringiem teraz. Dobrze myślę?